apps/api-wallet/src/app/transactions/transactions.controller.ts

```diff
@@ -32,9 +33,11 @@ export class TransactionsController {
   @UseGuards(AuthGuard('jwt'))
   @Get('get-transactions/:userID')
   getTransHistoryByUser(
-    @Param('userID', ParseIntPipe) userID: number
+    @Param('userID', ParseIntPipe) userID: number,
+    @Query('page') page: number = 1,
+    @Query('limit') limit: number = 3
```

Comment:
why is this 3?

Reply:
nag.set lang kog static value by 3 to get history by 3 for every request

Reply:
i see, create lang a dto for the pagination instead, add there the default value

Reply:
done